ersilia/cli/commands/api.py

```diff
@@ -1,49 +0,0 @@
-import click
```

Comment:
We should remove the file entirely.

Reply:
Thank you @DhanshreeA  for the feedback, This has now been done